### ISSUE TITLE
Allow providing custom defaults when registering command

### ISSUE THREAD

**Issue by axelboc** [narinluangrath/cypress-wait-for-stable-dom#4]
It would be nice to be able to pass custom defaults when registering the command:

```ts
registerCommand({ pollInterval: 300, timeout: 2000 });
```

**Comment by narinluangrath** [comment]
Fixed in version 0.0.4.

https://github.com/narinluangrath/cypress-wait-for-stable-dom#:~:text=The%20argument%20you%20provide%20to%20cy.waitForStableDOM%20will%20be%20deep%20merged%20with%20the%20following%20default%20object

**Comment by axelboc** [comment]
Hi @narinluangrath, thanks for looking into this but are you sure this is fixed? Basically what I'm after is something like this:

```ts
const DEFAULT_OPTIONS = {...} // as documented in README

const registerCommand = (defaultRegistrationOptions = {}) => {
  const defaultOptions = ... // merge `DEFAULT_OPTIONS` and `defaultRegistrationOptions `

  Cypress.Commands.add(
    'waitForStableDOM',
    { prevSubject: 'optional' },
    waitForStableDOMWithDefaults(defaultOptions)
  )
}

function waitForStableDOMWithDefaults(defaultOptions) {
  // Below is the current `waitForStableDOM` function, but `defaultOptions` now refers to the object passed during registration
  return (subject, userOptions = {}, iteration = 0) => {
    ...
  }
}

// Usage
registerCommand({ pollInterval: 300, timeout: 2000 });
cy.waitForStableDOM(); // no need to specify custom default options again
```

**Comment by narinluangrath** [comment]
@axelboc I totally misread the issue. I'll reopen it.

**Comment by narinluangrath** [comment]
Ok, _now_ it's fixed @axelboc (version `0.1.0`)

**Comment by axelboc** [comment]
I can confirm that it works great! Thanks a lot 😊
